save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -55,6 +56,19 @@ class DownloadFilesController(
         )
     }
 
+    /**
+     * @param owner owner name
+     * @return a image
+     */
+    @GetMapping("/api/avatar")
+    fun avatar(@RequestParam owner: String): ImageInfo? = additionalToolsFileSystemRepository.getAvatar(owner)?.let {
+        ImageInfo(
+            it.name,
+            Base64.getEncoder().encodeToString(FileUtils.getBytes(File(it.pathString))),
```

Comment:
I suggest to use `multipart/form-data` here, instead of manually loading files into memory. Take a look on how we send files from backend to preprocessor, for example.

Reply:
Fixed